app_widgetbook/lib/core/ui_elements/play_button.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:core/core.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook_annotation/widgetbook_annotation.dart';
+
+/// [PlayButton] default use case
+@WidgetbookUseCase(name: 'Default', type: PlayButton)
+Widget playButtonUseCase(BuildContext context) => PlayButton(
+      onPressed: () {},
+    );
```

Comment:
I know this is the use-case but I've seen that the `PlayButton` has a ripple effect. Is this intended?

Reply:
It's actually just to create the hover effect. But in general, I usually change the `Theme` wide `splashFactory` to disable ripple on iOS entirely as it's a material behavior. So I can do that in a separate PR as you said.